---
layout: research-hub
title: MBIPC Core
tag: cores
---

<aside id="lbrnEventSidebar">
  <button id="sideBug" class="btn" type="button" data-toggle="collapse" data-target="#lbrnEventSidebarUl" aria-expanded="false" aria-controls="collapseExample">
    <img src="{{site.baseurl}}/images/menuClosed.png" alt="Menu Bug">
  </button>
  <div id="lbrnEventSidebarUl">

    <div class="sidebar-holder">
      <div class="dot-text-holder">
        <li>Go To</li>
      </div>
    </div>
    <ul>

            <div class="sidebar-holder">
              <div class="dot-text-holder">
                <li class="active"><a href="#mbipccore">MBIPC</a></li>
              </div>
            </div>

            <div class="sidebar-holder">
              <div class="dot-text-holder">
                <li>
                  <a href="#trainingsupport">Core Personnel</a>
                </li>
              </div>
            </div>

            <div class="sidebar-holder">
              <div class="dot-text-holder">
                <li>
                  <a href="#genelab">GeneLab</a>
                </li>
              </div>
            </div>
            <div class="sidebar-holder">
              <div class="dot-text-holder">
                <li>
                  <a href="#ngs">NGS Services</a>
                </li>
              </div>
            </div>
            <div class="sidebar-holder">
              <div class="dot-text-holder">
                <li>
                  <a href="#singlecell">Single Cell Sequencing</a>
                </li>
              </div>
            </div>

    </ul>
  </div>
  <div class="">
    <li><a href="#">Top</a></li>
  </div>
</aside>
<div class="content-sidebar content">

  <section>
    <br>
    <!-- <h1 id="coreservices">Services</h1> -->
  </section>
  <div class="services">
    <div class="row">
      <div class="service main">
        <!-- <a href="bbc-core.html"> -->
        <div class="text">
          <h2 id="mbipccore">Molecular Biology and Immunopathology Core</h2>
          <p>The Molecular Biology and Immunopathology Core (MBIPC) and all 4 Center projects are located within the LSU SVM. MBIPC will be operationally headquartered within the existing administrative structure and laboratories of the Division of Biotechnology and Molecular Medicine (BioMMED).
            In addition, MBIPC will closely collaborate with the Center’s PCEC, also located within the SVM building. MBIPC will concentrate its support functions on molecular biology/recombinant DNA services and immunopathological characterization
            of in vitro and in vivo samples in conjunction with the SVM FACS Facility and the PBS Pathology Unit. Molecular biological services and reagents will include standard services such as DNA sequencing recombinant plasmid construction, cDNA
            library construction, and single cell DNA and RNA sequencing using the 10X genomics Chromium controller in conjunction with the Illumina Nextseq 550s/MiSeq sequencing equipment, for all 4 individual PJI’s projects. Advanced services will
            include processing and analysis of tissue/organ RNASeq experiments and single cell DNA and RNA sequencing data using advanced bioinformatics capabilities offered in conjunction with the Bioinformatics Unit of the LSU SVM LBRN/INBRE. In
            addition, MBIPC will offer real-time PCR analysis of mouse cytokines. The core has recently acquired the Agilent Seahorse XF platform which provides a direct measure of simultaneous measurements of OXPHOS and glycolytic rate, in live
            cells, in real-time. Using this technology, phenotypic evaluation of cancer cells in response to different metabolic substrates or inhibitors can be evaluated. Additional services include the production of monospecific and monoclonal
            antibodies, and coordination with the SVM Flow Cytometry Facility and the SVM Pathology Core offering advanced immunopathology and immunocytochemistry services. MBIPC has been established with strong experience in the application of
            advanced molecular and immunopathological expertise through coordinated access to existing SVM Facilities. New technological assays and methods will be adopted and/or developed over time based on individual project needs. </p>

        </div>
        <!-- <span> -->

        <!-- </span> -->
      </div>
    </div>
    <div class="row">
      <div class="service">
        <div class="text">
          <h2 id="trainingsupport">Key Core Personnel and Service </h2>
          <section class="directory Adcore peoplewrap">


            {% for people in site.data.CPCCR-adminCore %}
            {% include directory-about.html %}
            {% endfor %}

          </section>
          <p>Dr. Konstantin Kousoulas, (Director) will oversee all aspects of the scientific core, but specifically the core will rely on his expertise in gene expression in Molecular Biology and Infectious Diseases, where he has more than 30 years of experience. Dr. Kousoulas Is currently teaching a graduate bioinformatics course in advanced Transcriptomics attended virtually by more than 40 graduate students and faculty. The Co-Director, Dr. Vladimir Chouljenko’s expertise in RNASeq data analysis involving mouse and human genomes is essential where he has more than 28 years’ experience in Sequencing and recombinant DNA techniques. Dr. Antonieta Guerrero Plata (FACS, Facility Director) has more than 20 years of immunology experience and will oversee all immunological aspects of the Core and will facilitate access to the SVM FACS facility that is under her direction.  Dr. Ingeborg Langohr (board certified veterinary pathologist with more than 20 years of experience) will oversee and coordinate all pathological and immunopathological aspects of the work and facilitate access to the SVM Pathology and immune-histology cost center. Additional staff personnel include: Drs. Nithya Jambunathan and Muzammel Haque have more than 20 years of experience in molecular biology methodologies and gene expression patterns, as well as protein expression and purification, monoclonal and monospecific antibody production.
          </p>
          <!-- <a class="btn" title="Cores Contact" href="https://redcap.lbrn.lsu.edu/surveys/?s=JKJCL3QJ7E">Cores Contact</a> -->
        </div>
      </div>
    </div>
    <div class="row">
      <div class="service">
        <div class="text">
          <p>All BioMMED Core services are offered through a web-based date/time log sign on system (iLab), which is an effective set of tools for scheduling specific equipment, as well as ordering MBIPC services and supplies.  All Cores enlisted in this COBRE are subsidized by SVM. They are required to generate only 20% of Core expenses. It is envisioned that services ordered through the Center’s MBIPC will be subsidized at a level of up to 50% by the Core, while remaining charges will be on individual projects PIs.  Access to cost center services will be after review of the request by the Core leadership team and final approval by the AC. Additional information on these cores is listed below:</p>
        </div>
      </div>
    </div>
    <div class="row">
      <div class="service">
        <div class="text">
          <h2 id="genelab">GeneLab </h2>

          <p>GeneLab currently provides conventional and next generation sequencing (NGS) nucleic acid sequencing, recombinant DNA services (gene cloning, mutagenesis, and library construction), viral vector construction, quantitative PCR (qPCR) analysis, and reagents and supplies needed for biotechnology experiments. Research protocols and consultation for specific research projects are provided free of charge. The three qPCR machines located in GeneLab are utilized practically seven days per week (24/7). Training seminars and wet labs are provided by GeneLab staff to enable researchers to stay current in the rapidly changing field of molecular biology. MBIPC will work with the LBRN/INBRE and CEIDR-COBRE (Phase III-year 15)-funded GeneLab at SVM to provide Molecular Biology services to all Project and Pilot Project Leaders.
          </p>
          <!-- <a class="btn" title="Cores Contact" href="https://redcap.lbrn.lsu.edu/surveys/?s=JKJCL3QJ7E">Cores Contact</a> -->
        </div>
      </div>
      <div class="service">
        <div class="text">
          <h2 id="trainingsupport">GeneLab NGS and Bioinformatics Services </h2>

          <p>GeneLab bioinformatics functions and services are provided in a strategic alliance with the LSU Center for Computation and Technology (CCT) and the Louisiana Biomedical Research Network (LBRN) supported by an IDeA INBRE grant (P30 GM110760). This cross-utilization of manpower and funding sources provides significant capabilities. GeneLab’s infrastructure is composed of hardware and software tools and pipelines (Table 1). The GeneLab-Bioinformatics Unit operates three high-end work stations. The two visualization servers support windows, as well as Linux environments, providing memory intensive visualization tasks and tools working on both OS environments.  A computational server is equipped with large memory (128 GB) and large number of cores (32), is networked to the Ion PGM and Ion Proton platforms for dealing with analyses requiring large memory footprint and multi-core performance applications. These three high-end servers are backed with network-connected expandable NAS storage system, resulting in total storage space of 200 Terabytes at this time, which are directly connected to the servers and software tools integrated to the sequencing platforms.  An important aspect of our hardware and software infrastructure is that these computer storage systems are supported by the campus-wide high-performance computing (HPC) facilities managed by LSU’s Center for Computation & Technology (CCT). This also includes large HPC clusters operated by Louisiana Optical Network Initiative (http://www.loni.org). Computing infrastructure provided by CCT consists of a number of computing clusters and independent computing, application and database servers all connected to a high-speed campus network. Battery backup, fault monitoring, data snapshots, site-distinct mirroring, and off-site backups, are part of our comprehensive backup/recovery strategy and are all in place. Additionally, GeneLab provides access to the IBM Power8 Delta Cluster with a Configuration of total 16 compute nodes with 10-core 3.42 GHz Power8 processors, where 14 nodes are “thin” nodes with 256 GB memory  and 2 are “fat” nodes with I TB RAM each. 1/2 Petabyte storage space, with IBM’s state-of-the-art GPFS (General Parallel File System). Workflow pipelines can be created from existing software for Bioinformatics, Structural Modeling, and other analysis tasks using IBM’s SAC (Spectrum Application Center). SAC provides graphical user interface to create graph based customized workflow pipelines to users. This web-based real-time dashboard also provides for monitoring global workloads and resources, including resource usage reporting. Delta allocation is able to request through Delta webpage (https://delta.cct.lsu.edu) by creating a Delta account.  It supports big data analytics software such as Hadoop natively (fully optimized). Jobs can be submitted via a graphical Internet portal from anywhere on the LSU Campus.


          </p>
          <!-- <a class="btn" title="Cores Contact" href="https://redcap.lbrn.lsu.edu/surveys/?s=JKJCL3QJ7E">Cores Contact</a> -->
        </div>
      </div>
    </div>
    <div class="row">
      <div class="service" id="coreservices">
        <div class="text">
          <h2 id="ngs">NGS Services</h2>
          <p>Of particular interest to all COBRE investigators are GeneLab’s NGS support services in close collaboration with the LBRN Biocomputation, Biostatistics and Bioinformatics Core (BBBC) and the LSU Center for Computation and Technology (CCT). GeneLab is co-supported by the LBRN and CEIDR for the molecular cell biology, NGS services and bioinformatics services providing a reliable connection between the actual NGS experiments and the analysis of the NGS data. GeneLab’s NGS and bioinformatics support services became official on July 1, 2013 with the installation and training of personnel for the operation of two Ion Proton and Quant Studio Flex equipment to assist researchers at all stages of research, from experimental design to data generation and analysis. GeneLab has added the Illumina based Miseq and NextSeq sequencers in the summer of 2019. The Illumina MiSeq system is a next-generation, benchtop sequencer with a simple workflow, enabling researchers to move from sample to data in as little as eight hours. Capable of generating up to 15 Gb per run, the MiSeq system is the only next generation sequencer that integrates amplification, sequencing, and data analysis in a single instrument with a footprint of less than two square feet. GeneLab has also acquired an Illumina NextSeq 550 System. This sequencing system is the only desktop next-generation sequencing (NGS) system capable of sequencing a 30× human genome in a single run. Two flow cell formats and multiple reagent configurations enable 20–120 Gb of data output in a single run, providing flexibility across the broadest range of applications and study sizes. Additionally, Genelab has recently acquired the 10X Genomics Chromium Single Cell controller which will detect cell-to-cell gene expression variability and identify rare cell types in heterogeneous samples. Single-cell sequencing is a method that examines the genomes or transcriptomes of individual cells, providing a high-resolution view of cell-to-cell variation. Highly sensitive RNA sequencing (RNA-Seq) methods enable researchers to assess the individual contributions of single cells in complex tissues by profiling the transcriptome. The 10X Genomics Chromium Single Cell sequencing libraries have been best validated in the Illumina – NextSeq. The close alignment of GeneLab’s NGS capabilities and LSU CCTs bioinformatics capabilities provides an integrated approach to experimental design and quick analysis of the project data, while creating a unique forum for cross-training biologists and computer scientists in NGS methodologies and bioinformatics. Under this proposed Cancer COBRE, the expanded MBIPC Genelab will also focus on single cell trascriptomics and providing support for CRISPR/cas9 applications in conjunction with the PCEC. A variety of NGS equipment is currently available in GeneLab including two Ion Torrent PGM, two Ion Proton machines, and Quant Studio Flex (digital PCR validation). In addition, GeneLab has a variety of equipment to support NGS including qPCR machines and QuantStudio Flex systems; Fragment Analyzer from Advanced analytics to validated and quantify nucleic acid samples for NGS sample preparation; an EpMotion robotic arm for high throughput set-up and sampling; various computers and other equipment enabling quality control of nucleic acid samples; and preparing nucleic acid extractions from many samples at once. A visualization server supports Windows as well as Linux environments, providing memory intensive visualization tasks and tools working on both OS environments. A computational server equipped with large memory (128 GB) and large number of cores (32) is networked to the Ion PGM and Ion Proton platforms for dealing with analyses requiring large memory footprint and multi-core performance applications. These two high-end servers are backed with network-connected expandable NAS storage systems, resulting in total storage space of over 200 Terabytes at this time, which are directly connected to the servers and software tools integrated to the sequencing platforms. LSU-CCT staff work closely with GeneLab to transfer and analyze data and perform downstream custom-made bioinformatics analysis for individual projects (see below). </p>
          <!-- <a class="btn" title="BBC Software" href="downloads/HPC-Hardware-for-BBC.pdf" target="_blank">Hardware</a>
          <a class="btn" title="BBC Hardware" href="downloads/HPC-Software-List-for-BBC.pdf" target="_blank">Software</a> -->


        </div>
      </div>
    </div>
    <div class="row">
      <div class="service">

        <div class="text">
          <h2 id="singlecell">Single cell sequencing</h2>

          <p>A significant expansion of GeneLab research support has been achieved by launching single cell transcriptomic analysis. A major consideration for selection is the Chromium Single Cell Gene Expression Solution which provides a comprehensive, scalable solution for cell characterization and gene expression profiling of hundreds to tens of thousands of cells. It couples with the Illumina sequencers that GeneLab is going to acquire. The addition of the Single-Cell Isolator significantly expands our research capability at a minimal cost and builds on already existing expertise and equipment. The data analysis options provide sensitive and unbiased characterization of gene expression and is user-friendly. Single cell sequencing facilitates to: 1) Identify and characterize rare cell types; 2) Analyze and understand cellular heterogeneity and how this contributes to your biological system; 3) Perform cellular phenotyping with single cell RNA-seq to identify novel targets, biomarkers, and cell types and states without the need for pre-selected targets; 4) Evaluate mRNA and cell surface protein expression profiles within the same cell; 5) Perform high throughput and high resolution functional genetic; 6) Assess comprehensive gene expression phenotypes for individual CRISPR perturbations.</p>
          <!-- <a class="btn" href="http://www.cobre.ceidr.lsu.edu/genelab.html">GeneLab
          </a> -->
        </div>

      </div>
    </div>
  </div>
</div>
